starknet_py/net/full_node_client.py

```diff
@@ -93,6 +95,47 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
```

Comment:
I think it would be nice but I'm not sure if it can be handled in any logical way. Can these arguments be mixed like passing from_block as a number and `to_block` as a hash?

Reply:
I would stay with the current implementation, all methods in this class requiring block number or block hash have schema - they have two parameters `block_number` and `block_hash`, which are then passed to `get_block_identifier` method.